### ISSUE TITLE
Strip leading and tailing newlines from nodes in RPC replies

### ISSUE THREAD

**Issue by shermdog** [Juniper/py-junos-eznc#342]
RPCs introduced early into Junos have their data wrapped with newlines.

``` xml
<interface-information style="normal">
    <physical-interface>
        <name>\nge-0/0/0\n</name>
```

This makes searching for text difficult.

Proposed solution is to remote leading and trailing newlines from the node values by default and add an option to rpc() to disable.

Note: OpTables uses XMLs `normalize-space` on values by default.

**Comment by shermdog** [comment]
Decorator is in place, but no on by default.  This may be included in 2.0+

**Comment by stacywsmith** [comment]
This has been implemented in aa3d0d7. As stated, it is not enabled by default (916b4df), but it is available in releases >= 2.0
